ci/integration_tests/src/tests/cw20_stake_test.rs

```diff
@@ -0,0 +1,118 @@
+use core::time;
+use std::thread;
+
+use crate::helpers::{chain::Chain, helper::create_dao};
+use cosmwasm_std::{to_binary, Uint128};
+use cw20_stake::{msg::StakedValueResponse, state::Config};
+use cw_core_interface::voting::VotingPowerAtHeightResponse;
+use test_context::test_context;
+
+// #### ExecuteMsg #####
+
+#[test_context(Chain)]
+#[test]
+fn execute_stake_tokens(chain: &mut Chain) {
+    let voting_contract = "cw20_staked_balance_voting";
+
+    let res = create_dao(chain, None, "exc_stake_create_dao", chain.user.addr.clone());
```

Comment:
nit: maybe we can make the op_names like `"exc_stake_create_dao"` into constants for easier re-use?

Reply:
So, I wasnt planning on reusing them. These `op_name` strings are only for annotating the gas report instrumentation output with the name of the message that is associated with that gas cost.